design/host-cgroups.md

```diff
@@ -7,7 +7,11 @@
   - [`SandboxCgroupOnly` disabled (default, legacy)](#sandboxcgrouponly-disabled-default-legacy)
     - [What does this method do?](#what-does-this-method-do)
       - [Impact](#impact)
-  - [Summary](#summary)
+- [Supported cgroups](#supported-cgroups)
+  - [Cgroups V1](#cgroups-v1)
+  - [Cgroups V2](#cgroups-v2)
+    - [Limitations](#limitations)
```

Comment:
I wonder if "Distro Support" would make this section clearer as it isn't explaining a limitation of "cgroups v2", but more an issue with distro support of it.